src/tests/unit/test_tree_functionality/test_tree_node_helpers.py

```diff
@@ -0,0 +1,449 @@
+import unittest
```

Comment:
all testsare copied from the old "test_tree_node_helpers.py" file without being modified.